src/components/TeamMemberTasks/ReviewButton.jsx

```diff
@@ -34,55 +32,57 @@ const ReviewButton = ({
   }();
 
   const updReviewStat = (newStatus) => {
-    const resources = [...task.resources];
-    const newResources = resources.map(resource => {
-      const newResource = { ...resource, reviewStatus: newStatus };
-      newResource.completedTask = newStatus === "Reviewed";
-      return newResource;
-    });
-    const updatedTask = { ...task, resources: newResources };
+    const updatedTask = {
+      ...task,
+      resources: task.resources.map(resource => ({
+        ...resource,
+        reviewStatus: newStatus,
+        completedTask: newStatus === "Reviewed"
+      }))
+    };
     updateTask(task._id, updatedTask);
     setModal(false);
   };
 
   const buttonFormat = () => {
-    if (user.personId == myUserId && reviewStatus == "Unsubmitted") {
-      return <Button className='reviewBtn' color='primary' onClick={toggleModal} style={boxStyle}>
-        Submit for Review
-      </Button>;
-     } else if (reviewStatus == "Submitted")  {
-      if (myRole == "Owner" ||myRole == "Administrator" || myRole == "Mentor" || myRole == "Manager") {
+    if (reviewStatus === "Unsubmitted") {
+      if (user.personId === myUserId) {
+        return (
+          <Button className="reviewBtn" color="primary" onClick={toggleModal} style={boxStyle}>
+            Submit for Review
+          </Button>
+        );
+      }
+    } else if (reviewStatus === "Submitted") {
+      if (myRole === "Owner" || myRole === "Administrator" || myRole === "Mentor" || myRole === "Manager") {
```

Comment:
Nice job, however this could be refactored to: 

`myRole === "Owner" || "Administrator" || "Mentor" || "Manager"`

Reply:
I don't think this is correct since the javascript doesn't work like this. This expression will return "Administrator" or "Mentor" or "Manager" as long as the `myRole` is not "Owner". Consider `["Owner", "Administrator", "Mentor", "Manager"].includes(myRole)`

Reply:
Oh, I missed that part. 

Thanks for highlighting that.